Doc/library/getpass.rst

```diff
@@ -14,6 +14,7 @@
 
 The :mod:`getpass` module provides two functions:
```

Comment:
It doesn't really make much sense to have a colon here if the next items listed are not the referred-to functions. Instead, I suggest either rewriting this like, e.,g.

```suggestion
The :mod:`getpass` module provides functions to
get the current user's name and prompt them for a password.
```

***OR*** moving it below the ``include``:

```suggestion
```

(continued)

Reply:
I moved the include above the sentence.